package-lock.json

```diff
@@ -4418,6 +4418,13 @@
         "ini": "^1.3.4"
       }
     },
+    "global-registry-nodejs-client": {
+      "version": "github:CruGlobal/global-registry-nodejs-client#2d1d95e675432beaf5cc7c774617ffd1e9ae5e91",
+      "requires": {
+        "babel-eslint": "8.2.6",
```

Comment:
this reminds me --  https://github.com/CruGlobal/global-registry-nodejs-client/commit/af48b5a298c636b086a4183617d70d7dd6f7af99#r31231482

Reply:
Does not seem to be necessary, I have removed it from the source package